### ISSUE TITLE
SSL error in 2.3.6 

### ISSUE THREAD

**Issue by juliendehos** [NixOS/docker#21]
Hi, It seems there is a problem with the SSL config of the new image: 
```
error: unable to download 'https://github.com/NixOS/nixpkgs/archive/0e0df31.tar.gz': Problem with the SSL CA cert (path? access rights?) (77)
```

(see this gitlab-ci pipeline: https://gitlab.com/juliendehos/nixpkgs-tests/-/jobs/587075339)

**Comment by rycee** [comment]
I get the same issue, both for github.com and cachix.org:

- https://gitlab.com/rycee/nur-expressions/-/jobs/588465247
- https://gitlab.com/rycee/home-manager/-/jobs/588434988

**Comment by blitz** [comment]
@peti The reason is apparently this:

```
$ ls -l $NIX_SSL_CERT_FILE
ls: /nix/var/nix/profiles/default/etc/ssl/certs/ca-bundle.crt: No such file or directory
```

**Comment by parthy** [comment]
I see that the value of `NIX_SSL_CERT_FILE` has changed from
`/root/.nix-profile/etc/ssl/certs/ca-bundle.crt` in 2.3.4 to `/etc/ssl/certs/ca-certificates.crt` in 2.3.6. Could that be the reason?

**Comment by peti** [comment]
> Can this be researched further?

Yes, anybody who wants to is free to spend their time getting to the root of this change in behavior.
